src/SdkCommon/ClientRuntime.Azure.Authentication/ActiveDirectoryServiceSettings.cs

```diff
@@ -16,7 +16,7 @@ public sealed class ActiveDirectoryServiceSettings
 
         private static readonly ActiveDirectoryServiceSettings AzureSettings = new ActiveDirectoryServiceSettings
         {
-            AuthenticationEndpoint= new Uri("https://login.windows.net/"), 
+            AuthenticationEndpoint= new Uri("https://login.microsoftonline.com/"), 
```

Comment:
Please refer to @markcowl 's and my comments here: https://github.com/Azure/azure-sdk-for-net/pull/3030

login.microsoftonline.com is the new ARM endpoint that supports both the ARM and the Classic environment. As far as I know people are encouraged to use the new endpoint.